plasmapy/formulary/tests/test_collisions.py

```diff
@@ -1698,3 +1700,347 @@ def test_kwarg_method_error(self):
         """Testing kwarg `method` fails is not 'classical' or 'quantum'"""
         with pytest.raises(ValueError):
             coupling_parameter(self.T, self.n_e, self.particles, method="not a method")
+
+
+class TestCollisionFrequencies:
+    """Test the CollisionFrequencies class in collisions.py."""
+
+    attribute_test_case = CollisionFrequencies(
+        Particle("e-"),
+        Particle("e-"),
+        v_a=1 * u.m / u.s,
+        T_b=1 * u.K,
+        n_b=1 * u.m**-3,
+        Coulomb_log=1 * u.dimensionless_unscaled,
+    )
+
+    attributes_to_test = [
+        "slowing_down",
+        "transverse_diffusion",
+        "parallel_diffusion",
+        "energy_loss",
+        "v_0",
+    ]
+
+    return_values_to_test = [
+        "slowing_down",
+        "transverse_diffusion",
+        "parallel_diffusion",
+        "energy_loss",
+    ]
+
+    ones_array = np.ones(5)
+
+    @pytest.mark.parametrize("attribute_to_test", attributes_to_test)
+    def test_units(self, attribute_to_test):
+        """Test the return units"""
+
+        assert getattr(self.attribute_test_case, attribute_to_test).unit.is_equivalent(
+            u.Hz
+        )
+
+    @staticmethod
+    def get_limit_value(interaction_type, limit_type, frequencies):
+        """
+        Get the limiting values for frequencies given the two particles interacting, and their frequencies class.
+
+        These formulae are taken from page 31 of the NRL Formulary.
+        """
+
+        T_a = (frequencies.T_a * k_B).to(u.eV).value
+        T_b = (frequencies.T_b * k_B).to(u.eV).value
+
+        limit_values = []
+
+        if interaction_type == "e|e":
+            if limit_type == "slow":
+                limit_values.extend(
+                    [
+                        5.8 * 10**-6 * T_b ** (-3 / 2),
```

Comment:
FYI you can write scientific notation more concisely as `5.8e-6`. Also, -3/2 = -1.5 and etc. is also a little more compact since you eliminate the need for parentheses